### DISCUSSION TITLE
Add FreeCameraOptions to v10

### DISCUSSION THREAD

**Discussion by dpyeates** [rnmapbox/maps#2760] [Ideas]
## New Feature
Add options for accessing 'getFreeCameraOptions' of the Mapbox API which allows access to the properties of the underlying camera entity. Direct access to these properties allows more flexible and precise controlling of the camera. 

## Why
The FreeCamera allows precise positioning of the camera at a point location, pitch angle and bearing
This enables you to use the camera to 'fly' around the map.

Documentation:
https://docs.mapbox.com/mapbox-gl-js/api/properties/#freecameraoptions

Example:
https://docs.mapbox.com/mapbox-gl-js/example/free-camera-point/

**Comment by mfazekas** [comment]
Android: https://docs.mapbox.com/android/maps/api/10.9.1/mapbox-maps-android/com.mapbox.maps/-free-camera-options/
iOS: https://docs.mapbox.com/ios/maps/api/10.9.1/Classes/MapboxMap.html#/s:10MapboxMaps0A3MapC17freeCameraOptionsSo07MBMFreeeF0Cvp

**Comment by dpyeates** [comment]
Has there been any consideration to add this feature please? I have a use case that would be fantastic to be able to implement if I could 'fly' the camera around the map.

**Comment by dpyeates** [comment]
Any discussion at all on this? I really can't believe such a killer feature that is available hasn't yet been integrated into the React Native API?

**Comment by mfazekas** [comment]
@dpyeates Sorry we don't have many resources. We're welcome PR implement this feature, or you can consider sponsoring this feature,
